### PR TITLE
fix(doc): doc fix for vpc_routing_table_route next_hop

### PR DESCRIPTION
**Pull request by ujjwal-ibm**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

**Comment by ujjwal-ibm** [comment]
image

File: website/docs/r/is_vpc_routing_table_route.html.markdown [hunk 1]
```diff
@@ -65,7 +65,7 @@ Review the argument references that you can specify for your resource.
 - `action` - (Optional, String) The action to perform with a packet matching the route `delegate`, `delegate_vpc`, `deliver`, `drop`.
 - `destination` - (Required, Forces new resource, String) The destination of the route. 
 - `name` - (Optional, String) The user-defined name of the route. If unspecified, the name will be a hyphenated list of randomly selected words. You need to provide unique name within the VPC routing table the route resides in.
-- `next_hop` - (Required, String) The next hop of the route. It accepts IP address or a VPN connection ID. For `action` other than `deliver`, you must specify `0.0.0.0`. 
+- `next_hop` - (Required, String) The next hop of the route. It accepts IP address or a VPN gateway connection ID (`ibm_is_vpn_gateway_connection`) of a VPN Gateway (`ibm_is_vpn_gateway`) with the `mode = "route"` argument and in the same VPC as the route table for this route for an egress route. For action other than deliver, you must specify `0.0.0.0`.
```
Comment (SunithaGudisagarIBM1) [inline website/docs/r/is_vpc_routing_table_route.html.markdown:68]: Can you please test scenario mentioned in the documentation like specifying  "It accepts IP address or a VPN gateway connection ID (`ibm_is_vpn_gateway_connection`) of a VPN Gateway (`ibm_is_vpn_gateway`) with the `mode = "route"` argument and in the same VPC as the route table for this route for an egress route" and attach the result.

post the above comment ,PR is ready to go to upstream.

Thank you
Sunitha

Comment (ujjwal-ibm) [reply website/docs/r/is_vpc_routing_table_route.html.markdown:68]: `│ Error: next_hop must be omitted or specified as 0.0.0.0 when the action is delegate`


`│ Error: next_hop must be omitted or specified as 0.0.0.0 when the action is drop`

`│ Error: next_hop must be omitted or specified as 0.0.0.0 when the action is delegate_vpc`


```
"errors": [
            {
                "code": "validation_failed_oneof",
                "message": "Expected only one oneOf fields to be set: got 0",
                "target": {
                    "name": "RouteNextHopPrototype",
                    "type": "field"
                }
            },
            {
                "code": "validation_failed_required",
                "message": "RouteNextHopIP is invalid: Validation Failure for field: address",
                "target": {
                    "name": "RouteNextHopIP.address",
                    "type": "field"
                }
            },
            {
                "code": "validation_failed_oneof",
                "message": "VpnGatewayConnectionIdentity is invalid: Expected only one oneOf fields to be set: got 0",
                "target": {
                    "name": "VpnGatewayConnectionIdentity.VpnGatewayConnectionIdentity",
                    "type": "field"
                }
            },
            {
                "code": "validation_failed_max_length",
                "message": "VpnGatewayConnectionIdentity is invalid: VpnGatewayConnectionIdentityByID is invalid: Validation Failure for field: id",
                "target": {
                    "name": "VpnGatewayConnectionIdentity.VpnGatewayConnectionIdentityByID.id",
                    "type": "field",
                    "value": "0717-5f0c8a89-7898-4585-9e1f-b3b684597a97/0717-a931dbca-4bda-4300-875d-6a1bd023bccf"
                }
            },
            {
                "code": "validation_failed_required",
                "message": "VpnGatewayConnectionIdentity is invalid: VpnGatewayConnectionIdentityByHref is invalid: Validation Failure for field: href",
                "target": {
                    "name": "VpnGatewayConnectionIdentity.VpnGatewayConnectionIdentityByHref.href",
                    "type": "field"
                }
            }
```

**Comment by SunithaGudisagarIBM1** [comment]
@ujjwal-ibm 
This PR is approved from my end.
Thank you

**Review by SunithaGudisagarIBM1 — APPROVED** [review]
@ujjwal-ibm
This PR is approved from my end.
Thank you

**Comment by ujjwal-ibm** [comment]
moving to upstream

**Pull request closed without merging**